packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -78,12 +78,25 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
         showErrorIcon={showErrorIcon} />
     );
 
-    let renderChildren = () => (
-      <Flex direction="column" UNSAFE_className={classNames(labelStyles, 'spectrum-Field-wrapper')}>
-        {children}
-        {hasHelpText && renderHelpText()}
-      </Flex>
-    );
+    let renderChildren = () => {
+      if (labelPosition === 'side') {
```

Comment:
We used to have it similar to this, https://github.com/adobe/react-spectrum/commit/0795617214074cd999358bf9a42fe4da80e2c2df#diff-abb1c606574077fc353b2ee07124e2a936b3a7a32ae8fe4ff471b65be704887dR80

which was turned into  the current state via this PR https://github.com/adobe/react-spectrum/commit/e80b387035826f3ebfd0d0cb3a0b51337c20fbb8

Can we make sure we aren't losing focus again?

this might be a good one for a test

Reply:
Ok, had some time to do more than glance at this, I see the difference, the previous one switched on hasHelpText, this only changes based on labelPosition, which is unlikely to change while the user is focused here. I'm adding a test for the focus lost anyways, just so we don't get bitten by it in the future